src/network/networkd-state-file.c

```diff
@@ -3,6 +3,8 @@
 #include <netinet/in.h>
 #include <linux/if.h>
 
+#include "dns-resolver-internal.h"
+
```

Comment:
as mentioned elsewhere, we don't separate these out. just put them along with the other `""` includes, alphabtically sorted